ground/lib/station.js

```diff
@@ -91,11 +92,8 @@ export class Station extends EventEmitter {
 
   checkRadio() {
     return new Promise((resolve, reject) => {
-      if (this.radioDevice || this.mock) {
-        resolve(this.radioDevice);
-        return;
-      }
-      this.detectRadio(resolve, reject);
+      resolve();
```

Comment:
we should be making sure the RTLSDR is connected somehow in this function instead of making it an empty stub. I'm open to punting on that until this pull request is merged though, provided you create a bug to make sure we don't forget :)